src/pages/Controllers/_controllers.scss

```diff
@@ -80,6 +88,8 @@
     }
 
     .p-form-help-text {
+      margin-top: 0;
```

Comment:
This is causing a large gap between inputs and help text. I think [Vanilla default](https://vanillaframework.io/docs/base/forms#stacked) looks better here.

Reply:
I agree